src/graphics/tooltip.c

```diff
@@ -312,7 +312,7 @@ void tooltip_handle(const mouse *m, void (*func)(tooltip_context *))
         reset_timer();
         return;
     }
-    tooltip_context context = {m->x, m->y, 0, 0, 0, 0, 0, 0};
+    tooltip_context context = {m->x, m->y, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
Do we still need the extra zeroes here?
(aside: do we need the zeroes at all? I seem to remember that when you initialize at least one variable of a struct, the rest of it is set to 0 automatically)

Reply:
> (aside: do we need the zeroes at all? I seem to remember that when you initialize at least one variable of a struct, the rest of it is set to 0 automatically)

Answer: Indeed, no.

From [Stack Overflow](https://stackoverflow.com/questions/37548287/partially-initializing-a-c-struct):

> This is guaranteed as per section 6.7.8 of the C standard (with highlighting specific to this case):
> 
>
> > 10 If an object that has automatic storage duration is not initialized explicitly, its value is indeterminate. If an object that has static storage duration is not initialized explicitly, then:
> > 
> > —if it has pointer type, it is initialized to a null pointer;
> >
> > —if it has arithmetic type, it is initialized to (positive or unsigned) zero;
> > 
> > —if it is an aggregate, every member is initialized (recursively) according to these rules;
> > 
> > —if it is a union, the first named member is initialized (recursively) according to these rules.
> > 
> > ...
> > 
> > 21 If there are fewer initializers in a brace-enclosed list than there are elements or members of an aggregate, or fewer characters in a string literal used to initialize an array of known size than there are elements in the array, the remainder of the aggregate shall be initialized implicitly the same as objects that have static storage duration.

I'll get rid of those zeroes.